src/modules/DeleteTodo.js

```diff
@@ -0,0 +1,21 @@
+export default function DeleteTodo(toDo) {
+  const checkbox = document.querySelectorAll('.checkbox');
+  const button = document.querySelector('.clear');
+
+  for (let i = 0; i < checkbox.length; i += 1) {
+    checkbox[i].addEventListener('change', (e) => {
+      if (e.target.checked) {
+        toDo.list[i].completed = true;
+      } else {
+        toDo.list[i].completed = false;
+      }
+    });
+  }
```

Comment:
It's better to use forEach loop and ternary operator, it makes your code more clean